RZTouchID/RZTouchID.h

```diff
@@ -57,15 +156,35 @@ OBJC_EXTERN NSString* const kRZTouchIDErrorDomain;
 /**
  *  Whether TouchID is currently available and configured with at least one finger and
  *  a passcode. If this method returns NO, then attempts to retrive passwords from the keychain using TouchID will fail.
+ *
+ *  @return Returns true if a passcode is set, the device supports touch ID
  */
 + (BOOL)touchIDAvailable;
 
 /**
+ *  Whether TouchID is currently available and configured with at least one finger and
+ *  a passcode. If this method returns NO, then attempts to retrive passwords from the keychain using TouchID will fail.
+ *
+ *  @param userID A string representing a user ID. This could be an email, a UUID or a username.
+ *
+ *  @return Returns true if a passcode is set, the device supports touch ID and the userID provided has a password stored in the keychain.
+ */
+- (BOOL)touchIDAvailableForIdentifier:(NSString *)userID;
+
+/**
  *  Return the RZTouchID object initialized with the service name prefix used to create unique keychain keys.
  *
  *  @param servicePrefix A unique name representing this login service in the keychain.
  */
-- (instancetype)initWithKeychainServicePrefix:(NSString *)servicePrefix;
+- (instancetype)initWithKeychainServicePrefix:(NSString *)servicePrefix touchIDMode:(RZTouchIDMode)touchIDMode;
```

Comment:
Might want to change "touchIDMode" to "authenticationMode" or something to indicate that the mode is related to authentication